pages/blog/dont-forget-your-vpn.mdx

```diff
@@ -0,0 +1,46 @@
+---
+title: "Don’t Forget Your VPNs and Tracker Blockers"
+date: "2021-01-27"
+author: "Andy Jih"
+pullQuote: "Ad blockers are great, until they’re not"
+image: "dont-forget-your-vpn/intercom_blocked.png"
+tags: [product, notes]
+---
+
+I was testing the new [Intercom](https://intercom.com) plugin that our team had built, and ran into a strange snag. When adding apps to Grouparoo, we have this handy “Test Connection” button that checks to make sure the connection details and credentials work. When I was testing this new Intercom plugin, something strange appeared this time around.
+
+<Image
+  centered
+  alt="Steps of a simple sync"
+  src="dont-forget-your-vpn/intercom_blocked.png"
+  width={845}
+  height={586}
+/>
+
+I was testing locally, but the Test Connection button actually talks to an external API, so I was confused that the connection failed due to something from localhost.
```

Comment:
Add some text that described what is shown in the image, "The connection didn't work!" (for vision-impared people, RSS readers, etc)

Reply:
Oops, forgot to update the Alt texts